src/app/sort/sort.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit, Output } from "@angular/core";
+import { EventEmitter } from "@angular/core";
+
+interface Options {
+  label: string;
+  value: string;
+}
+
+@Component({
+  selector: "app-sort",
+  templateUrl: "./sort.component.html",
+  styleUrls: ["./sort.component.scss"],
+})
+export class SortComponent {
+  sortSelected: string = "Sort by";
+  sortOptions = [
```

Comment:
Do you think it makes sense for this component to have the options as input?
(think about reusability and the values)